### ISSUE TITLE
Eval with official checkpoint gets better depth prediction than reported in paper.

### ISSUE THREAD

**Issue by synsin0** [GANWANSHUI/GaussianOcc#6]
Great thanks for your great work and immediate open-source. When I eval with official checkpoint, I find the rayiou and miou metric match the reported in paper while the depth prediction (AbsRel, SqRel, RMSE) is better than reported. I generate depth in the same way of your script. I'd like to ask if my log is normal so that your depth prediction is better than other methods. Attached is my log: 
[log.txt](https://github.com/user-attachments/files/16745248/log.txt)

**Comment by synsin0** [comment]
I think one possible reason is that the model uses gt_pose when evaluation, but the gt_pose isn't used in the table, but I'm not quite sure how to disable gt_pose.

**Comment by GANWANSHUI** [comment]
Hi, thanks for the question. the result is normal. The depth result with semantic training is better than the non-semantic, no matter the gt pose or the proposed learned pose. The depth result in Table 3 is no semantic training and the network is adapted from [Simpleocc](https://github.com/GANWANSHUI/SimpleOccupancy) for fair comparison, which has been stated in the paper. I still did not release the pretrained model in Table 3. 

Besides, the gt pose definition is a little different, please see [here](https://github.com/GANWANSHUI/GaussianOcc/blob/2a6a9f3f35c96b67024791f0eb4d929f4f3dc3a5/options.py#L340).

**Comment by synsin0** [comment]
Thanks for your answer! I have another question regarding the stage 2 training. This setting does not use auxiliary frames from past or future to training, which means in stage 2 the supervision comes only from current frame. Is it right? Does stage 1 training involves multi-frame multi-view stereo? Or where does the model learn 3D geometry (I may think 3D geometry learns from multi-view stereo but the overlap of one-frame multi-view is limited.) May you explain a little bit?

**Comment by GANWANSHUI** [comment]
Yes, the supervision comes only from the current frame. we did not use the auxiliary frames as in OccNeRF. We used to try but without observing the improvement. Actually, OccNeRF only uses the auxiliary frames for depth estimation. 

Stage 1 also follows the setting as surrounddepth without involving multi-frame multi-view stereo.

**Comment by synsin0** [comment]
If all training are with single-frame and the extrinsics between multi-cameras are known, what does predicted pose stand for, and why we need to predict a pose? Is it the relative pose between two adjacent frames as two-frame Structure-from-Motion in SurroundDepth? 

My last question is that in mono_dataset.py the inputs['all_cam_center'] is hard coded, is it the parameter of nuScenes or DDAD dataset? I don't find elsewhere to change this value. But this value is used to relocate voxel center in vox_grid used in 3dgs.

**Comment by GANWANSHUI** [comment]
The single-frame supervision (3 frames) is compared with the auxiliary frames (5 frames) proposed in OccNeRF, we still require the self-supervised monocular depth estimation pipeline to construct the single-frame supervision (depth map warpping and synthesis). The predicted relative pose is the temporal pose from the sequence image, not the spatial pose from the camera extrinsic. 

inputs['all_cam_center']: this parameter has not been used, we could ignore it.

**Comment by synsin0** [comment]
Thanks for your patient answers! I have no more questions now.

**Comment by synsin0** [comment]
Sorry for the very last question. Just to check this line (https://github.com/GANWANSHUI/GaussianOcc/blob/2a6a9f3f35c96b67024791f0eb4d929f4f3dc3a5/runner.py#L930)，Does it mean that if we set self.opt.gt_pose = yes, we do not use gt pose; if we set self.opt.gt_pose= no ; we use gt pose. The yes/no is contrary to whether to use gt_pose.

**Comment by GANWANSHUI** [comment]
Yes, sorry for the idiot setting, I may modify it in the future.
